core/chains/evm/config/toml/defaults/Ethereum_Mainnet.toml

```diff
@@ -1,5 +1,6 @@
 ChainID = '1'
 LinkContractAddress = '0x514910771AF9Ca656af840dff83E8264EcF986CA'
+FinalityDepth = 10
```

Comment:
This is wrong.
Ethereum mainnet shouldn't be touched.

Reply:
Added FinalityDepth = 50 in Ethereum_Mainnet.toml and set FinalityDepth = 10 in fallback.toml